js/ui/lookup.js

```diff
@@ -294,7 +302,7 @@ const Lookup = DropDownList.inherit({
 
                     closeOnOutsideClick: true,
 
-                    popupWidth: (function() { return $(this.element()).outerWidth(); }).bind(this),
+                    popupWidth: (function() { return this._getPopupWidth(); }).bind(this),
```

Comment:
It seems we should change the description of [popupWidth](https://js.devexpress.com/Documentation/20_1/ApiReference/UI_Widgets/dxLookup/Configuration/#popupWidth) option

Reply:
A technical writer will fix after merge